cluster/ci/monitoring/ghproxy.json

```diff
@@ -0,0 +1,801 @@
+{
```

Comment:
NIP for now: Give it a try with mixins and jsonnet?

Reply:
can be done in follow-up?

Reply:
Sure.